02/app.js

```diff
@@ -1,2 +1,23 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+function sum() {
+  for(let i = 1; i <= x; i++) {
```

Comment:
Czasami na tym etapie trzeba wyjaśnić po kolei jak działa pętla for, opowiedzieć każdy etap iteracji - kiedy jest sprawdzany warunek, kiedy jest uruchamiana zawartość, a kiedy ikrementacja itp.